cvat-ui/src/components/annotation-page/canvas/views/canvas2d/canvas-wrapper.tsx

```diff
@@ -481,12 +481,15 @@ class CanvasWrapperComponent extends React.PureComponent<Props> {
         }
 
         if (prevProps.highlightedConflict !== highlightedConflict) {
-            const severity: HighlightSeverity | null =
-                highlightedConflict?.severity ? (highlightedConflict?.severity as any) : null;
-            const highlightedElementsIDs = highlightedConflict?.annotationConflicts.map(
-                (conflict: AnnotationConflict) => conflict.clientID,
-            );
-            canvasInstance.highlight(highlightedElementsIDs || null, severity);
+            const severity: HighlightSeverity | undefined = highlightedConflict
+                ?.severity as unknown as HighlightSeverity;
+            const highlightedClientIDs = (highlightedConflict?.annotationConflicts || [])
+                .map((conflict: AnnotationConflict) => annotations
+                    .find((state) => state.serverID === conflict.serverID && state.objectType === conflict.type),
```

Comment:
`conflict.type` has a string type,  looks we need to change it to `ObjectType`

Reply:
Done